### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by mmarkelov**
Update packages and fix "Deprecation warning for String based operators"
https://github.com/sequelize/sequelize/issues/8417

### CODE REVIEW

**Comment by jorrit** [comment]
react is at 16.3.1 now. Can you update the PR?

**Comment by langpavel** [comment]
Thank you @mmarkelov

**Comment by damiangreen** [comment]
This commit has broken the react-intl branch. It can no longer be merged because of conflicts in package.json and yarn.lock. I imagine that branch will also need updating.

File: src/data/sequelize.js [hunk 1]
```diff
@@ -7,10 +7,11 @@
  * LICENSE.txt file in the root directory of this source tree.
  */
 
-import Sequelize from 'sequelize';
+import Sequelize, { Op } from 'sequelize';
 import config from '../config';
 
 const sequelize = new Sequelize(config.databaseUrl, {
+  operatorsAliases: Op,
```
Comment (SystemDisc) [inline src/data/sequelize.js]: This should be set to `false` and not `Op`.

**Pull request merged**